### DISCUSSION TITLE
links/

### DISCUSSION THREAD

**Discussion by giscus** [Cgfyufsygsm/cgfyufsygsm.github.io#61] [Announcements]
# links/

我们到底在追求些什么

https://blog.imyangty.com/links/

**Comment by elainafan** [comment]
在树洞认出tygg了，来加下友链~
title: "Elainafan's blog",
intro: "只因"现在"深爱其中",
link: "https://www.elainafan.one/",
avatar: "https://www.elainafan.one/avatars/elainafan.jpg"

**Comment by hanlife02** [comment]
tygg，我的博客头像信息需要更新一下，谢谢你
```
{
    avatar: "https://hanlife02.com/avatar.svg"
}
```

**Reply by Cgfyufsygsm** [reply]
已更新～

**Comment by SunXT-0719** [comment]
我最近也搞了个homepage，求加个友链喵
{
    title: "SunXT",
    intro: "SunXT's 网络漫游",
    link: "https://sunxt-0719.github.io/",
    avatar: "https://sunxt-0719.github.io/assets/images/avatar.jpg"
}

**Reply by Cgfyufsygsm** [reply]
已更新～
